index.html

```diff
@@ -672,14 +678,20 @@ <h3>Controllers</h3>
 }
             </pre>
 
-            <p class="note" title="Authorization vs authentication">
-Note that authorization provided by the value of `controller` is
-separate from authentication as described in Section [[[#authentication]]].
-This is particularly important for key recovery in the cases of cryptographic key
-loss, where the [=subject=] no longer has access to their keys, or cryptographic
-key compromise, where the [=controller=]'s trusted third parties need to
-override malicious activity by an attacker. See [[[#security-considerations]]]
-for information related to threat models and attack vectors.
+            <p class="note" title="Authentication versus Authorization">
+Note that the definition of [=authentication=] is different from the definition
+of [=authorization=]. Generally speaking, [=authentication=] answers the
+question of "Who is this?" while [=authorization=] answers the question of "Are
+they allowed to perform this action?". The `authentication` property in this
```

Comment:
```suggestion
of [=authorization=]. Generally speaking, [=authentication=] answers the
question of "What entity is trying to act?" while [=authorization=] answers the
question of "Are they allowed to perform this action?". The `authentication`
property in this
```

Reply:
I'm not sure it's better to change "Who is this?" to "What entity is trying to act?" -- as it implies that there is an additional action beyond authentication itself. Maybe we can interpret that in some some vacuously true way, but I don't think we should imply that authenticating involves additional actions. Authentication could just be a response to "Is this you?" -- with no further action implied by the entity that answers.

Reply:
@dlongley — Maybe this?
```suggestion
of [=authorization=]. Generally speaking, [=authentication=] answers the
question of "What entity is trying to act?" while [=authorization=] answers the
question of "Are they allowed to perform this action?" Note that the action being
attempted might simply be proving the entity's own identity, i.e., authenticating.
The `authentication` property in this
```

Reply:
Maybe "What entity am I interacting with?" This removes the requirement that the other entity is the one driving the action (which may not be the case).

Reply:
It strikes me that perhaps your suggestion is be driven by an effort to reconcile the language in the two questions together in some way. However, the two questions are not necessarily linked and can be asked independently of one another entirely. One does not necessarily need to know "who" someone is for them to act/be authorized, nor does one necessarily need to be "authorized to act" (nor "acting" at all) in order for them to be authenticated.